app/models/comment.rb

```diff
@@ -179,16 +179,20 @@ def readable_publish_date
     end
   end
 
-  def remove_notifications
-    Notification.remove_all_without_delay(notifiable_ids: id, notifiable_type: "Comment")
-  end
-
   private
 
+  def index_id
```

Comment:
you'll notice a pattern around the code of me moving inside `private` things that are only called inside the model itself